hpOneView/resources/facilities/power_devices.py

```diff
@@ -47,14 +47,227 @@ def __init__(self, con):
+        body. The response contains the power delivery device resource as added to the appliance with default and
+        assigned properties expanded. The id and uri are assigned by the management appliance and are used to uniquely
+        identify this particular resource.
+
+        Args:
+            resource: power device information
+            timeout: Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: added power device.
+        """
+        return self._client.create(information, timeout=timeout)
+
+    def remove(self, resource, force=False, timeout=-1):
+        """
+        Deletes the set of power-devices according to the specified parameters. A filter is required to identify the
+        set of resources to be deleted. The actual deletion will proceed asynchronously. Use the returned task resource
+        to track the completion and any errors that may occur.
+
+        Args:
+            resource: dict object to remove
+            force:
+                 If set to true the operation completes despite any problems with
+                 network connectivity or errors on the resource itself. The default is false.
+            timeout: Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns: Result status
+        """
+        return self._client.delete(resource, force=force, timeout=timeout)
+
+    def add_ipdu(self, information, timeout=-1):
+        """
+        Add an HP iPDU and bring all components under management by discovery of its management module. Bring the
+        management module under exclusive management by the appliance, configure any management or data collection
+        settings, and create a private set of administrative credentials to enable ongoing communication and management
+        of the iPDU. Use "force" to claim the device, even if claimed by another management appliance
+
+        Args:
+            resource: power device information
+            timeout: Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: added power device.
+        """
+        uri = self.URI + "/discover"
+        return self._client.create(information, uri=uri, timeout=timeout)
+
+    def update(self, resource, timeout=-1):
+        """
+        Updates the resource for the specified {id}. The properties that are omitted (not included as part of the the
+        request body) are reset to their respective default values. The id and uuid properties are required and cannot
+        be changed. To update existing {rest-resource} first perform a GET request to retrieve the current properties,
```

Comment:
I think we should rephrase this comment, since the user don't need to perform the GET/PUT requests. Those operations could be made internally through the SDK, calling the get and update methods.